import_export/forms.py

```diff
@@ -1,7 +1,7 @@
 import os.path
+from collections.abc import Iterable
```

Comment:
I don't think this change is necessary because the import relates to a typing declaration [here](https://github.com/django-import-export/django-import-export/blob/5d9915c708dbefe148bd74acaffefd9dfbf891aa/import_export/forms.py#L115).